project_name/utils/blocks.py

```diff
@@ -11,7 +11,7 @@
 from {{ project_name }}.utils.struct_values import CardStructValue, LinkStructValue
 
 
-class AccordionBlock(blocks.StructBlock):
+class AccordionSectionBlock(blocks.StructBlock):
```

Comment:
Great catch! The original code seems really odd with its redefinition of `AccordionBlock`...

Reply:
Thank you!